deps/rabbitmq_prometheus/src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -433,7 +433,8 @@ membership(Pid, Members) when is_pid(Pid) ->
 membership({Name, Node}, Members) ->
     case Node =:= node() of
         true ->
-            case is_process_alive(whereis(Name)) of
+            Pid = whereis(Name),
+            case is_pid(Pid) andalso is_process_alive(Pid) of
```

Comment:
the is_pid/1 check is enough to determine liveness so the `is_process_alive/1` call isn't necessary